deeprvat/deeprvat/config.py

```diff
@@ -319,7 +324,7 @@ def update_config(
             else:
                 logger.info("Not performing EAC filtering of baseline results")
             logger.info(f"  Correcting p-values using {correction_method} method")
-            alpha = config.get("alpha_seed_genes", config.get("alpha"))
+            alpha = config["baseline_results"].get("alpha_seed_genes", config.get("alpha"))
```

Comment:
I think alpha is not present at the top level of the config any more so ` config.get("alpha")` won't work. I think it must be `config["evaluation"]["alpha"]`